features/bootstrap/FeatureContext.php

```diff
@@ -91,6 +91,7 @@ public function thereIsANamed($entity, $name)
             }
 
             if ('user' === $entity) {
+                throw new \Exception("Je m'aprête à créer un user parce qu'il n'y en a pas");
```

Comment:
The Exception will interrupt the script and the next line won't be reached.

Reply:
I totally forgot to remove it, sorry ! I do it right now.